vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/SpaceLink.java

```diff
@@ -15,6 +16,7 @@
 import edu.asu.diging.vspace.core.model.ISpaceLink;
 
 @Entity
+@Table(name = "spacelink")
```

Comment:
why are you changing the table name? that would probably make all existing space links not show up anymore.